packages/hooks/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@leafygreen-ui/hooks",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Kit Custom Hooks",
+  "main": "./dist/index.js",
+  "scripts": {
```

Comment:
Don't forget to add the following script:
```
"ts:emit": "tsc --project ./tsconfig.json"
```

Reply:
👍